src/utils/mailup.ts

```diff
@@ -218,6 +227,8 @@ function toMailupAddress(
 export function MailUpTransport(
   options: IMailUpTransportOptions
 ): nodemailer.Transport {
+  const fetchAgent =
+    options.fetchAgent !== undefined ? options.fetchAgent : fetch;
```

Comment:
what's 'fetch' in this context, node-fetch ? (is it imported ?)

Reply:
you are right, I think it's just exported by `lib.dom`, not real